packages/fxa-settings/src/components/ConnectedServices/index.tsx

```diff
@@ -123,9 +83,10 @@ export const ConnectedServices = () => {
               {...{
                 key: client.clientId,
                 name: client.name,
+                deviceType: client.deviceType,
                 location: client.location,
                 lastAccessTimeFormatted: client.lastAccessTimeFormatted,
-                canSignOut: false,
+                canSignOut: true,
```

Comment:
Hmm, it looks like this is defaulting `canSignOut` to true for all clients. Is that true, though? Can we log out any RP?

Reply:
This was the ask in the jira issue, https://jira.mozilla.com/browse/FXA-1757. We have https://jira.mozilla.com/browse/FXA-1760 for the signout flow.